addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3312,34 +3312,6 @@ public Action:MakeBoss(Handle:timer, any:boss)
 		}
 	}
 
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable_demoshield"))!=-1)
-	{
-		if(IsBoss(GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")))
-		{
-			TF2_RemoveWearable(client, entity);
-		}
-	}
-
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_usableitem"))!=-1)
```

Comment:
Crashing because taunting? I see many bosses can use taunts flawlessly. I know that TauntEm plugin uses ```tf_wearable_vm``` entity and set the item definition index to force taunt somehow with engine call. What to do?